client/cypress/integration/pages/flightEdit.spec.js

```diff
@@ -0,0 +1,50 @@
+describe("check flight page", () => {
+  const flightId = 6;
+
+  before(() => {
+    cy.seedDb();
+    cy.visit("/");
+    cy.loginNormalUser();
+    cy.visit(`/flug/${flightId}`);
+  });
+
+  it("Check presence and function of edit button", () => {
+    cy.get("#flight-details");
+    cy.get("button").contains("Flug bearbeiten").click();
+    cy.url().should("include", `/flug/${flightId}/bearbeiten`);
+  });
+
+  const oldGliderId = "cd25b974-1e30-4969-ba46-34990461990d";
+  const newGliderId = "8f48aa72-6ea0-477e-ae3c-e76fa99e7fb5";
+
+  // const oldFlightReport = "eius ullam omnis nesciunt amet dolorem";
+  const newFlightReport = "Foo";
+
+  it("check edit flight details", () => {
+    cy.get("#glider-select").should("have.value", oldGliderId);
```

Comment:
Testcases müssen immer alleine lauffähig sein.

In diesem Testcase muss zwangsläufig der Testfall davor gelaufen sein. Sonst findet er den select nicht.